src/commands/react-native/upload.ts

```diff
@@ -193,6 +193,9 @@ export class UploadCommand extends Command {
 
   private getPayloadsToUpload = async (useGit: boolean): Promise<RNSourcemap[]> => {
     const payloads = await this.getMatchingRNSourcemapFiles()
+    if (payloads.length > 1) {
+      this.context.stdout.write('Upload to Datadog of multiple sourcemaps is not yet officially supported!')
```

Comment:
Is it possible for this condition to be true? `getMatchingRNSourcemapFiles` always returns one `RNSourcemap`.

Reply:
For now it is not.
I decided to keep this code as we are going to support multiple files upload soon, and I believed there were more chances to introduce bugs by removing the multi-file logic temporarily than by keeping it.

The goal of this comment is to warn devs that would like to patch this repo to implement it, but I can remove it.